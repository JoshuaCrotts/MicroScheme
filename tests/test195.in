;
; Random number generation with set seed
;
(random-set-seed! 1337439)
(do ((i 1 (+ i 1)))
    ((> i 10) (display "done"))
    (random-integer 1 100))