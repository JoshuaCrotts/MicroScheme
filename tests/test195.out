random-set-seed!
47
72
27
22
88
8
16
21
44
98
done
